### PR TITLE
Make a table instead of using a list

### PR DESCRIPTION
**Pull request by plaindocs**
When we merge #1002 we can use similar CSS

- [x] Fix list in `pages/agent/v3/help/_artifact_search.md`
- [x] Use new CSS

Output

image

instead of this

image

### CODE REVIEW

File: pages/agent/v3/help/_artifact_search.md [hunk 1]
```diff
@@ -46,33 +46,52 @@ The above will return a list of filenames separated by newline.
...
+<tr><td><code>--endpoint value</code></td><td><p>The Agent API endpoint (default: "<code>https://agent.buildkite.com/v3</code>") [<code>$BUILDKITE_AGENT_ENDPOINT</code>]</p></td>
+<tr><td><code>--no-http2</code></td><td><p>Disable HTTP2 when communicating with the Agent API. [<code>$BUILDKITE_NO_HTTP2</code>]</p></td>
+<tr><td><code>--debug-http</code></td><td><p>Enable HTTP debug mode, which dumps all request and response bodies to the log [<code>$BUILDKITE_AGENT_DEBUG_HTTP</code>]</p></td>
+<tr><td><code>--no-color</code></td><td><p>Don't show colors in logging [<code>$BUILDKITE_AGENT_NO_COLOR</code>]</p></td>
+<tr><td><code>--debug</code></td><td><p>Enable debug mode [<code>$BUILDKITE_AGENT_DEBUG</code>]</p></td>
+<tr><td><code>--experiment value</code></td><td><p>Enable experimental features within the buildkite-agent [<code>$BUILDKITE_AGENT_EXPERIMENT</code>]</p></td>
+<tr><td><code>--profile value</code></td><td><p>Enable a profiling mode, either cpu, memory, mutex or block [<code>$BUILDKITE_AGENT_PROFILE</code>]</p></td>
+</table>
```
Comment (plaindocs) [inline pages/agent/v3/help/_artifact_search.md:96]: Fix this :-p

**Comment by plaindocs** [comment]
Can be improved further, but instead of this

image


we now have this

image

**Review by karensawrey — APPROVED** [review]
Love it!

**Comment by plaindocs** [comment]
Update, fixed some spacing

image

can't track down the end visual glitch

image

**Pull request merged**